src/LibraryManager.Vsix/Json/Completion/LibraryIdCompletionProvider.cs

```diff
@@ -77,17 +76,26 @@ protected override IEnumerable<JSONCompletionEntry> GetEntries(JSONCompletionCon
 
                 if (set.Completions != null)
                 {
+                    bool versionSpecific = false;
+                    bool setVersionSpecific = true;
+
                     foreach (CompletionItem item in set.Completions)
                     {
+                        if (setVersionSpecific)
+                        {
+                            versionSpecific = item.DisplayText.Contains("@");
+                            setVersionSpecific = false;
+                        }
```

Comment:
Could the LibraryIdCompletionEntry class do this for you?

Reply:
Yes. This can be moved just using DisplayText

---
In reply to: [198552083](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198552083) [](ancestors = 198552083)